test/gtest/common/test_helpers.h

```diff
@@ -162,6 +162,27 @@
 namespace ucs {
 
 extern const double test_timeout_in_sec;
+extern const double watchdog_timeout_default;
+
+typedef struct {
+    pthread_t         thread;
+    pthread_mutex_t   mutex;
+    pthread_cond_t    cv;
+    double            timeout;
+    pthread_t         parent_thread;
```

Comment:
minor: rename 'parent_thread' to 'watched_thread' or 'main_tread'

Reply:
fixed